app/src/sharedTest/java/org/oppia/app/utility/OrientationChangeAction.kt

```diff
@@ -0,0 +1,51 @@
+package org.oppia.app.utility
+
+import android.view.View
+import androidx.test.espresso.UiController
+import androidx.test.espresso.ViewAction
+import androidx.test.espresso.matcher.ViewMatchers.isRoot
+import org.hamcrest.Matcher
+import android.content.pm.ActivityInfo
+import android.app.Activity
+import android.content.Context
+import android.content.ContextWrapper
+import android.view.ViewGroup
+
+// https://gist.github.com/nbarraille/03e8910dc1d415ed9740
+/** This class helps in managing orientation changes in test-cases which are visible on device too. */
+class OrientationChangeAction(private val orientation: Int) : ViewAction {
+
+  companion object {
+    fun orientationLandscape(): ViewAction = OrientationChangeAction(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE)
```

Comment:
Let's make these file-level functions instead of companion functions. Then we can make this action private to the file to make things a bit cleaner.

Reply:
I am actually in favour of keeping this implementation because this makes the usage of these functions quite easy in test cases. 

sample:

`onView(isRoot()).perform(orientationLandscape())`